docs/current_docs/adopting.mdx

```diff
@@ -62,7 +62,7 @@ When choosing the language to use for your pipeline, follow these principles:
 
 - *Optimize for participation*. The more people on the team are likely to participate in the development of the pipeline, the better.
 
-- *Check SDK availability*. Dagger supports 3 stable SDKs (Python, Go, Typescript), but it also has several community-contributed SDKs, some of which are actively used in production despite not being officially supported. Take the time to investigate the status of available SDKs, and ask the community for their opinion. Our Discord server is a good place to start - each supported language has a dedicated channel.
+- *Check SDK availability*. Dagger supports 3 stable SDKs (Python, Go, Typescript), but it also has several community-contributed SDKs, some of which are actively used in production despite not being officially supported. Take the time to investigate the status of [available SDKs](https://github.com/dagger/dagger/tree/main/sdk), and ask the community for their opinion. Our Discord server is a good place to start - each supported language has a dedicated channel.
```

Comment:
suggest replacing community-contributed with "experimental" 

also change 

> each supported language has a dedicated channel. 

to 

> each language has a dedicated channel 

because I think we have channels for even the experimental SDKs

Reply:
@mircubed do you have time to make these changes on your fork, i can also do it in a separate PR if youd like.